plugin.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# name: discourse-group-tag-associations
+# about: Associate groups with tags and include tagged topics in group lists.
+# version: 1.0
+# authors: pmusaraj
+# url: https://github.com/discourse/discourse-group-tag-associations
+
+enabled_site_setting :group_tag_associations_enabled
+
+PLUGIN_NAME ||= 'DiscourseGroupTagAssociations'
+
+  %w[
+    ../app/models/group_tag_association.rb
+  ].each { |path| load File.expand_path(path, __FILE__) }
+
+after_initialize do
+  require_dependency File.expand_path('../app/controllers/groups_controller.rb', __FILE__)
+
+  Group.class_eval do
```

Comment:
Can we not use the helpers for this? `add_to_class` and so forth? Those are generally better because you get the auto reloading behavior.